### DISCUSSION TITLE
How to change cursor style according to zoom

### DISCUSSION THREAD

**Discussion by GoldenRhynoDev** [igordanchenko/yet-another-react-lightbox#286] [Q&A] [answered]
If I do action like zoom in, I want to change the cursor style to "zoom in ".
Also like zoom out.
How can I change the cursor style smoothly?
Thanks.

**Comment by igordanchenko** [comment, marked as answer]
I'm afraid there is no built-in option to change the cursor based on the zoom level. 

Please keep in mind that there is usually more than one zoom-related action that you can perform, so changing the cursor to a single action may be misleading. Here are the actions that you can perform based on zoom level:

1) Minimum zoom level (initial, no zoom) <---- that's the only case with a single zoom-related action
- zoom-in 

2) Medium zoom level
- zoom-in
- zoom-out
- click-and-drag

3) Maximum zoom level
- zoom-out
- click-and-drag

**Comment by GoldenRhynoDev** [comment]
Hi @igordanchenko 
https://github.com/igordanchenko/yet-another-react-lightbox/assets/121535863/5d0cd51f-2156-43c3-a706-14ecf0e28b03

As you can see this video, the cursor not change correctly.
How can I make the changing cursor style according to zoom in or zoom out?
Have you got any idea?

**Reply by igordanchenko** [reply]
As I explained above, this library doesn't support what you are asking for.
